tomcat/src/main/docker/Dockerfile

```diff
@@ -34,7 +34,10 @@ RUN mkdir -p $CATALINA_BASE/webapps \
     && chown -R tomcat:tomcat ${CATALINA_BASE} \
     && chmod -R 400 $CATALINA_BASE/conf
 
+# Set path where apps should be added to the container
+ENV APP_DESTINATION ROOT.war
```

Comment:
`${CATALINA_BASE}/webapps/ROOT.war` would be more appropriate, if we decide later to change the `WORKDIR`